frontend/src/pages/AboutUs.jsx

```diff
@@ -70,14 +69,13 @@ function AboutUs() {
             isClicked.Marcelo ? "displayOn" : "displayOff"
           }`}
         >
-          Marcelo, c'est le génie du backend. Il navigue dans les bases de
-          données et les serveurs avec une aisance déconcertante. Pour lui, la
-          sécurité et la stabilité sont les maîtres mots de tout développement
-          web. Il aime résoudre les problèmes complexes, que ce soit en
-          optimisant les requêtes SQL ou en mettant en place des systèmes de
-          sauvegarde impeccables. Marcelo est le gardien des données, veillant à
-          ce que tout fonctionne parfaitement en coulisses, permettant ainsi aux
-          utilisateurs de profiter d'une expérience web sans faille.
+          Marcelo, this is the genius of the backend. He navigates the basics of
+          data and servers with disconcerting ease. For him, the security and
+          stability are the key words of all development web. He enjoys solving
+          complex problems, whether in optimizing SQL queries or setting up
+          monitoring systems impeccable backup. Marcelo is the data keeper,
+          ensuring ensures that everything works perfectly behind the scenes,
+          allowing users to enjoy a seamless web experience.
```

Comment:
Cadeau => Marcelo is the genius of data management. He navigates databases and servers with disarming ease. For him, security and stability are the watchwords of any web development. He enjoys solving complex problems, whether it's optimizing SQL queries or implementing flawless backup systems. Marcelo is the guardian of data, ensuring that everything runs seamlessly behind the scenes, allowing users to enjoy a seamless web experience.